docs/topics/modules/glossary.adoc

```diff
@@ -19,11 +19,12 @@ This file is consumed for the automatic generation of infotips used by OSIO web
   // endterm
 
 * When adding a new term, either use the `uuidgen` command on the CLI or use  https://www.uuidgenerator.net/ to generate a new UUID.
+Note: Planner UX provides uuids for each of the terms requiring infotips, use them rather than generating new ones.
```

Comment:
Some of the UUIDs come from the space template and UX might not know about those UUIDs upfront.